doc/user/source/referenz/items/standard_attribute/eval.rst

```diff
@@ -238,6 +243,50 @@ angegeben werden sondern können sich relative auf andere Items beziehen.
     - eval_trigger: **sh.** my.item | **sh.** my.other.item
 
 
+Attribut *eval_on_trigger_only*
+===============================
+
+Mit dem Attribut **eval_on_trigger_only** wird die Auswertung des **eval** Ausdrucks
+übersprungen, wenn dessen Auswertung nicht durch einen definierten **eval_trigger**
+erfolgt ist.
+
+.. code-block:: yaml
+
+   Raum:
+
+       Temperatur:
+           type: num
+           name: average temperature
+           eval: avg
+           eval_trigger:
+             - room_a.temp
+             - room_b.temp
+
+       Temperatur2:
+           type: num
+           name: average temperature
+           eval: avg
+           eval_on_trigger_only: true
+           eval_trigger:
+             - room_a.temp
+             - room_b.temp
+
+.. note::
+
+    Wichtig anzumerken ist, dass die Nutzung der Funktionen, die **eval_trigger**
+    Werte zur Berechnung verwenden (sum, avg, min, max, and und or) eingeschränkt
+    wird.
```

Comment:
Was genau heißt _eingeschränkt_? Geht oder geht nicht?

Reply:
Das ist im folgenden Beispiel beschrieben - bei Aktivierung über Trigger bleibt es unverändert, bei Zuweisung eines Wertes wurde bisher die "Triggerauswertung" durchgeführt, jetzt wird der Wert direkt zugewiesen (was sicher eher unerwünscht ist, aber der Vorgabe "eval_on_trigger_only" entspricht)

Reply:
Achso - es wird nur eingeschränkt, wenn `eval_on_trigger_only` auf True gesetzt ist. Ohne das Attribut gibt es keine Änderungen zu vorher.

Reply:
Habs in der Doku ergänzt